addons/ekco/0.10.2/reboot/shutdown.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
This file changed lines 7-14 and 22-26. There were no other changes since 0.10.1. If this PR is approved I'll push a new 0.10.2 tag for ekco identical to 0.10.1 before merging.